backend/analytics_server/mhq/store/repos/code.py

```diff
@@ -347,6 +347,22 @@ def get_team_repos(self, team_id) -> List[OrgRepo]:
         team_repo_ids = [tr.org_repo_id for tr in team_repos]
         return self.get_repos_by_ids(team_repo_ids)
 
+    @rollback_on_exc
+    def get_prs_merged_without_review(self, team_id, interval) -> List[PullRequest]:
+        AllOrg = self.get_team_repos(team_id)
+        AllOrg_ids = [tr.id for tr in AllOrg]
+        return (
+            self._db.session.query(PullRequest)
+            .filter(PullRequest.repo_id.in_(AllOrg_ids))
+            .filter(PullRequest.merge_time == None)
+            .filter(
+                PullRequest.created_in_db_at.between(
+                    interval.from_time, interval.to_time
+                )
+            )
```

Comment:
@Kamlesh72 created_at_in_db is the timestamp at which the data was stored in our db, which is irrelevant for a user. If they wish to query data in a date range, they are likely searching PRs that were either created or updated in that interval.

Reply:
@samad-yar-khan created_at & updated_at should fix this

Reply:
@Kamlesh72 there is one more logical bug  in this query. Its fetching open PRs as well as merged PRs.

Reply:
@samad-yar-khan I believe this is for @RISHIKESHk07

Reply:
@samad-yar-khan querying state for merged should correct that , i did try testing on a team repo but i see only closed & merged state Prs ,but  we do have a few open Prs , (selcted pr merge while creating team did i miss anything ??)